interview-questions.md

```diff
@@ -8,62 +8,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: A parameter is the broader term that refers to what types of data a function can take in if the function requires any. An argument is the specific value that is passed as the parameter for a given function.
 
-  Researched answer:
+  Researched answer: A parameter is a placeholder variable that can be named anything but for the sake of efficiency, the parameter should be named something descriptive and related to the function. The parameter will always be within the parentheses of the function expression. An argument, however, is the actual value of what is passed to the function within the function invocation.
```

Comment:
Such a close confusing relationship, that many developers use interchangeably. You are correct parameters are placeholders and arguments are the values